net.natesales.Aviator.yml

```diff
@@ -0,0 +1,61 @@
+app-id: net.natesales.Aviator
+command: aviator
+sdk: org.gnome.Sdk
+runtime: org.gnome.Platform
+runtime-version: "42"
+
+finish-args:
+  - --share=ipc
+  - --device=dri
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --filesystem=host
+
+cleanup:
+  - /include
+  - /lib/pkgconfig
+  - /man
+  - /share/doc
+  - /share/gtk-doc
+  - /share/man
+  - /share/pkgconfig
+  - '*.la'
+  - '*.a'
+
+modules:
+  - name: svt-av1
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - "-DCMAKE_BUILD_TYPE=Release"
+      - "-DBUILD_SHARED_LIBS=ON"
+      - "-DBUILD_APPS=OFF"
+      - "-DBUILD_DEC=ON"
+      - "-DBUILD_ENC=ON"
+      - "-DBUILD_TESTING=OFF"
+    cleanup:
+      - "/include"
+      - "/lib/pkgconfig"
+    sources:
+      - type: git
+        url: https://gitlab.com/AOMediaCodec/SVT-AV1.git
+        tag: v1.2.1
+        commit: 6e24fb7f5f7438bb04061dc1a159f0c5b882d847
+
+  - name: ffmpeg
+    config-opts:
+      - "--enable-libdav1d"
+      - "--enable-libsvtav1"
+      - "--enable-libopus"
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://ffmpeg.org/releases/ffmpeg-5.1.tar.xz
+        sha256: 55eb6aab5ee235550fa54a33eaf8bf1b4ec66c01453182b12f6a993d75698b03
+
+  - name: aviator
+    builddir: true
+    buildsystem: meson
+    sources:
+      - type: dir
```

Comment:
This needs to point to some external git repository.

Reply:
Oops, fixed!